extension/source/components/Button.tsx

```diff
@@ -1,52 +1,26 @@
+import { IconProps } from 'phosphor-react';
 import * as React from 'react';
 
 const Button = (props: {
+  className?: string;
```

Comment:
I realized that the same button component will be reused with variation at different places so I moved the styling outside such that it is implemented in parent where button is used.